### DISCUSSION TITLE
My account was auto flagged when I bought Github Team

### DISCUSSION THREAD

**Discussion by k4chalov** [community/community#49289] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Actions are disabled, I can't deploy. My team does not have access to repository, because I can't purchase them additional seats. I can't migrate code back to personal account - that was also disabled.
And the best part is, support doesn't answer for 6 days.
I think GitHub should change something in launching organization.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there 👋🏼 

We appreciate your patience - Suppport answers tickets based on their resources and the amount of tickets ahead of yours. The Community is unable to assist in these matters, so I'll be locking this post. Cheers!
